cassovary-core/src/main/scala/com/twitter/cassovary/graph/DynamicDirectedGraphHashMap.scala

```diff
@@ -46,17 +46,18 @@ abstract class DynamicDirectedGraphHashMap(val storedGraphDir: StoredGraphDir)
    * Get the total number of edges in the graph. It's a very slow function. Use it carefully.
    */
   def edgeCount = iterator.foldLeft(0) {(accum, node) =>
-    accum + node.inboundCount  + node.outboundCount
+    accum + (storedGraphDir match {
+      case OnlyOut | BothInOut => node.outboundCount
+      case OnlyIn => node.inboundCount
+      case Mutual => 2 * node.outboundCount // count each edge twice
```

Comment:
Shouldn't this be just outboundCount, since every edge is anyway being counted twice (once for each endpoint)?